pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -700,6 +700,20 @@ type KubeAPIServerConfig struct {
 	// Defaults to 1h.
 	// +optional
 	EventTTL *metav1.Duration `json:"eventTTL,omitempty" protobuf:"bytes,12,opt,name=eventTTL"`
+	// Logging contains configuration for the logs level and http access logs
+	// +optional
+	Logging *Logging `json:"logging,omitempty" protobuf:"bytes,13,opt,name=logging"`
+}
+
+// Logging contains configuration for the logs level and http access logs
+type Logging struct {
+	// Verbosity is the kube-apiserver log verbosity level
+	// Defaults to 2.
+	// +optional
+	Verbosity *int32 `json:"verbosity,omitempty" protobuf:"bytes,1,name=verbosity"`
+	// HTTPAccessLogs is the kube-apiserver access logs level
+	// +optional
+	HTTPAccessLogs *int32 `json:"httpAccessLogs,omitempty" protobuf:"bytes,2,name=httpAccessLogs"`
```

Comment:
Both are of type `int32`, so I assume the second attribute also describes a log verbosity? Should we rename it accordingly (`HTTPAccessVerbosity`)?